challenges/index.html

```diff
@@ -9,7 +9,7 @@
 
   <script src="objects-arrays.js"></script>
   <script src="functions.js"></script>
-  <script src="prototypes.js"></script>
+<!--   <script src="prototypes.js"></script> -->
   <script src="classes.js"></script>
 </head>
 
```

Comment:
- Good job on the sprint challenge Inaki. You have succesfully completed the MVP and even tackled a stretch goal 

- All of Your solutions to the assignment look great. 

- You git workflow is fluid. You made frequent commits with descriptive commit messages.

- You are really improving everyday, keep up the good work 👍